lib/config/decrypt.spec.ts

```diff
@@ -1,11 +1,19 @@
+import crypto from 'node:crypto';
 import { Fixtures } from '../../test/fixtures';
 import { CONFIG_VALIDATION } from '../constants/error-messages';
-import { decryptConfig } from './decrypt';
+import { Json } from '../util/schema-utils';
+import { toBase64 } from '../util/string';
+import { decryptConfig, tryDecryptEcdhAesGcm } from './decrypt';
 import { GlobalConfig } from './global';
+import { EcJwkPriv } from './schema';
 import type { RenovateConfig } from './types';
 
 const privateKey = Fixtures.get('private.pem');
 const privateKeyPgp = Fixtures.get('private-pgp.pem');
+// renovate private key
+const privateEc = Fixtures.get('private-ec.json');
+const renovatePrivKey = Json.pipe(EcJwkPriv).parse(privateEc);
```

Comment:
```suggestion
const renovatePrivateKey = Json.pipe(EcJwkPriv).parse(privateEc);
```

I think we should spell it out in full. 😉